plugins/net_plugin/net_plugin.cpp

```diff
@@ -4317,7 +4317,8 @@ namespace eosio {
            "   myprod,myhostname.com:9876,198.51.100.1\n"
            "   myprod,myhostname2.com:9876,[2001:0db8:85a3:0000:0000:8a2e:0370:7334]"
            )
-         ( "agent-name", bpo::value<string>()->default_value("Vault Agent"), "The name supplied to identify this node amongst the peers.")
+         ( "agent-name", bpo::value<string>()->default_value("Spring {version}"),
```

Comment:
Should we append `Spring version` to the agent name if the user does not use the default?

Reply:
I don't think so. If someone doesn't want the version in their `agent-name` then they can just leave off `{version}`.

Reply:
https://github.com/AntelopeIO/spring/pull/1549/commits/79c63901f50221a306a08aae6f3e5bcbe7309b76